src/main/java/com/launchacademy/programmingswaglist/controllers/ProductRestController.java

```diff
@@ -55,12 +57,17 @@ public ProductRestController(ProductRepository productRepository, CategoryReposi
   }
 
   @GetMapping("/api/v1/products")
-  public Iterable<Product> getList(Pageable pageable){
-    return productRepository.findAll(pageable);
+  public Iterable<Product> getList(){
+    return productRepository.findAll();
   }
 
   @GetMapping("/api/v1/products/{productId}/show")
   public Product getProduct(@PathVariable Integer productId) {
     return productRepository.findById(productId).orElseThrow(() -> new ProductNotFoundException());
   }
+
+  @PostMapping("/api/v1/products")
```

Comment:
this endpoint should consider an invalid product. Can we update to run validations and return a 422 if the product is invalid?